storage/testdb/testdb.go

```diff
@@ -114,7 +115,7 @@ func newEmptyDB(ctx context.Context) (*sql.DB, func(context.Context), error) {
 	}
 
 	db.Close()
-	db, err = sql.Open("mysql", mysqlURI()+"/"+name)
+	db, err = sql.Open("mysql", mysqlURI()+name)
```

Comment:
how about using https://golang.org/pkg/path/filepath/#Join?

Reply:
Seems good, but I don't know how compatible the "file paths" are with MySQL URIs. Leaving as is for now.

Reply:
ok, maybe add a TODO() for checking usage of `filepath.Join`?

and here, you may want to add a trailing `/` after the `name` as well, if `sql.Open` requires it.

Reply:
`sql.Open` requires it only for when the database name is not included into the URI (see line 104, for instance).